compiler/qsc_project/tests/tests.rs

```diff
@@ -36,6 +36,7 @@ fn basic_manifest() {
                     ),
                     license: None,
                     language_features: [],
+                    lints: [],
```

Comment:
Does this need to be specified if you're not specifying anything? Ideally not, I assume.

Or if this is NOT testing actually settings in qsharp.json take effect, we should add tests for that.

Reply:
Just added a unit-test to `language_service/src/state/tests.rs` to make sure the lints_config are getting loaded from the manifest.